lib/util/PDFDocumentWithTables.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const PDFDocument = require('pdfkit')
+
+/**
+ * This class extends the pdfkit functionality to provide a tables method, which
+ * will write tableData (2-d array) to the pdf doc.
+ *
+ * See https://www.andronio.me/2017/09/02/pdfkit-tables/ for more info.
```

Comment:
I'd recommend saying somewhere in the comments that this code was copied from this source.